### PR TITLE
Kelvinj54 editing exercise

### PR DESCRIPTION
**Pull request by kelvinj54**
## Description:
- **What's changed?** Changes for editing exercise
- **Is this PR related to a Monolith release?** 

### Resolves:

* [OKTA-XXXXXX](https://oktainc.atlassian.net/browse/OKTA-XXXXXX)

### CODE REVIEW

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-basic-sign-in/main/index.md](https://github.com/okta/okta-developer-docs/blob/b4ff6ceb72aecbdd21fb518e868d2ed2969f7532/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Foie-embedded-sdk-use-case-basic-sign-in%2Fmain%2Findex.md) | [93](https://okta.acrolinx.cloud/api/v1/checking/scorecards/eda73dc9-0625-40fa-879c-bf795e30ed35) | 100  | 100  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-basic-sign-in/main/nodejs/getuserprofile.md](https://github.com/okta/okta-developer-docs/blob/b4ff6ceb72aecbdd21fb518e868d2ed2969f7532/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Foie-embedded-sdk-use-case-basic-sign-in%2Fmain%2Fnodejs%2Fgetuserprofile.md) | [100](https://okta.acrolinx.cloud/api/v1/checking/scorecards/81adfd38-0328-49e4-bae8-e288bfb27e4c) | 100  | 100  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-basic-sign-in/main/nodejs/integrationsteps.md](https://github.com/okta/okta-developer-docs/blob/b4ff6ceb72aecbdd21fb518e868d2ed2969f7532/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Foie-embedded-sdk-use-case-basic-sign-in%2Fmain%2Fnodejs%2Fintegrationsteps.md) | [89](https://okta.acrolinx.cloud/api/v1/checking/scorecards/fd4895c2-d75a-489b-a4f8-a1aefa1e9464) | 64  | 64  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-basic-sign-in/main/nodejs/pwdoptionalusecase.md](https://github.com/okta/okta-developer-docs/blob/b4ff6ceb72aecbdd21fb518e868d2ed2969f7532/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Foie-embedded-sdk-use-case-basic-sign-in%2Fmain%2Fnodejs%2Fpwdoptionalusecase.md) | [100](https://okta.acrolinx.cloud/api/v1/checking/scorecards/b395e5ad-f6d0-4895-92ec-9ad2f2ee0289) | 100  | 100  | 100  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.013d8817-f7e1-444c-b41f-52d8e785f7ef?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-basic-sign-in/main/nodejs/integrationsteps.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ### Your app displays the sign-in page
 
-Build a sign-in page that captures both the user's name and their password.
+Build a sign-in page that captures both the `username` and `password`.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-basic-sign-in/main/nodejs/integrationsteps.md:3]: This isn't talking about an actual field name. It's telling the dev to build a sign-in page that captures a user's name and a user's password.

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-basic-sign-in/main/nodejs/integrationsteps.md [hunk 2]
```diff
@@ -15,7 +15,7 @@ Source image: https://www.figma.com/file/YH5Zhzp66kGCglrXQUag2E/%F0%9F%93%8A-Upd
 
 ### The user submits their username and password
 
-When the user submits their `username` and `password`, pass them as parameters to [`OktaAuth.idx.authenticate()`](https://github.com/okta/okta-auth-js/blob/master/docs/idx.md#idxauthenticate).
+Once the user submits their `username` and `password`, pass them as parameters to [`OktaAuth.idx.authenticate()`](https://github.com/okta/okta-auth-js/blob/master/docs/idx.md#idxauthenticate).
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-basic-sign-in/main/nodejs/integrationsteps.md:18]: Once can suggest simultaneous action. We need to eliminate this type of ambiguity for translations.

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-basic-sign-in/main/nodejs/integrationsteps.md [hunk 3]
```diff
@@ -24,11 +24,11 @@ const transaction = await authClient.idx.authenticate({ username, password });
 
 ### Your app handles an authentication success response
 
-`authenticate()` returns a `transaction` object with a `status` property indicating the current state of the sign-in flow. Handle the returned `IdxStatus` value accordingly:
+`authenticate()` returns a `transaction` object with a `status` property, indicating the current state of the sign-in flow. Handle the returned `IdxStatus` value accordingly:
 
 #### Success status
 
-When the user correctly supplies their password, `IdxStatus` equals `IdxStatus.SUCCESS`. Call `tokenManager.setTokens()` to save the tokens retrieved from the response for future requests, and then redirect the user back to the home page. The user is now signed in.
+Once the user correctly enters their `password`, `IdxStatus` equals `IdxStatus.SUCCESS`. Call `tokenManager.setTokens()` to save the tokens retrieved from the response for future requests, and then redirect the user back to the home page. The user is now signed in.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-basic-sign-in/main/nodejs/integrationsteps.md:31]: Once can suggest simultaneous action. We need to eliminate this type of ambiguity for translations.

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-basic-sign-in/main/nodejs/pwdoptionalusecase.md [hunk 4]
```diff
@@ -1,2 +1,2 @@
 >
-> To learn about a sign-in use case where the password is optional, see the [Sign in with email only](/docs/guides/pwd-optional-sign-in-email/nodeexpress/main/) guide.
+> To learn about a sign-in use case where the password is optional, refer to the [Sign in with email only](/docs/guides/pwd-optional-sign-in-email/nodeexpress/main/) guide.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-basic-sign-in/main/nodejs/pwdoptionalusecase.md:2]: InfoDev style is to use "see"

**Review by susanharper-okta — COMMENTED** [review]
comments

**Pull request closed without merging**